packages/react-app/src/index.jsx

```diff
@@ -5,52 +5,22 @@ import { ChakraProvider, ColorModeScript } from "@chakra-ui/react";
 import { CacheProvider } from "@emotion/react";
 import { EmotionCache } from "@emotion/cache";
 import "@fontsource/space-mono";
+import "@fontsource/inter";
 import "@fontsource/poppins";
 import { mode } from "@chakra-ui/theme-tools";
 
 import ReactDOM from "react-dom";
 import App from "./App";
 import "./index.css";
 import createEmotionCache from "./styles/createEmotionCache";
-// import customTheme from "./styles/customTheme";
+import { colorModeConfig, theme } from "./styles/customTheme";
```

Comment:
Worth noting: the colors used were _slightly_ toned down from the designed theme, but this update corrects that